console/console-init/ui/src/Routes.tsx

```diff
@@ -18,6 +18,8 @@ const getConnectionDetail = () =>
   import("modules/connection-detail/ConnectionDetailPage");
 const getDeviceDetailPage = () =>
   import("modules/iot-device-detail/DeviceDetailPage");
+const getIoTProjectDetail = () =>
```

Comment:
can we have the name cosistency as we have Page postfix. 
getIoTProjectDetailPage